packages/world/ts/config/v2/worldWithShorthands.ts

```diff
@@ -54,7 +54,7 @@ export type validateNamespacesWithShorthands<namespaces, scope extends Scope = A
   [namespace in keyof namespaces]: {
     [key in keyof namespaces[namespace]]: key extends "tables"
       ? validateTablesWithShorthands<namespaces[namespace][key], scope>
-      : validateNamespaces<namespaces[namespace], scope>[key];
+      : validateNamespace<namespaces[namespace], scope>[key];
```

Comment:
might be nice to add some regression tests to make sure this doesn't slip through the cracks again!

Reply:
Agreed!

Reply:
Agreed!

Would we just test that each namespace in a shorthand config is valid?

Reply:
I think a type+runtime test for a valid+invalid namespace should be good

Reply:
Only thing is we can't test until `namespaces` is enabled in https://github.com/latticexyz/mud/pull/2880

Reply:
why? there's already tests for namespaces in `worldWithShorthands.test.ts`

Reply:
Doesn't that mean that the `namespaces` key is enabled on `defineWorldWithShorthands`? That feels incorrect